### PR TITLE
Clean and lint license

### PR DESCRIPTION
**Pull request by rtritto**
This change is [Reviewable](https://reviewable.io/reviews/mongo-express/mongo-express/1045)
- - -

Continue #1044
Related #213

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,21 +1,10 @@
...
+Copyright (c) 2012 Chun-hao Hu
+Copyright (c) 2016-present Multiple Contributors
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```
Comment (shakaran) [inline LICENSE.md:6]: Are you sure that avoid the 80 chars wordwrap is the suitable here?

File: LICENSE.md [hunk 2]
```diff
@@ -1,21 +1,10 @@
 MIT License
 
-Copyright (c) 2012  Chun-hao Hu Copyright (c) 2016-present Multiple Contributors
+Copyright (c) 2012 Chun-hao Hu
+Copyright (c) 2016-present Multiple Contributors
```
Comment (shakaran) [inline LICENSE.md:4]: I am not sure if multiple contributors should be put in more than one line for linters? What command as linter did you use?

File: README.md [hunk 3]
```diff
@@ -339,18 +339,3 @@ Here is an example of a document which can be read/edited in mongo-express (medi
       "func": Code(function() { alert('Hello World!') }),
       "symbol": Symbol("test")
     }
-
```
Comment (shakaran) [inline README.md:342]: Probably good to add a link reference to the file, so easily the people can check it

**Review by shakaran — CHANGES_REQUESTED** [review]

File: LICENSE.md (hunk 1 above)
Comment (rtritto) [reply LICENSE.md:6]: You're right, I will restore it.

File: LICENSE.md (hunk 2 above)
Comment (rtritto) [reply LICENSE.md:4]: I only restored from _README_.
Anyway multiple copyrights must be placed on new line and maybe separated by empty line.
We can read [this](https://github.com/cs3org/.github).

File: README.md (hunk 3 above)
Comment (rtritto) [reply README.md:342]: I think no (I checked some famous projects). License can be viewed in About section of Github.

**Pull request merged**